packages/boilerplate/theme/plugins/boilerplate.js

```diff
@@ -1,4 +1,4 @@
 import { setup } from '@vue-storefront/boilerplate-api';
+import { config } from './boilerplate-config';
```

Comment:
How about keeping it in one file (config in `boilerplate.js`) or making a folder to group those files? It doesn't seem right for me to have a file that is not a plugin in `plugins` dir (i know we have it in CT integration - I'm also thinking about a better way to structure this)

Reply:
👍